### DISCUSSION TITLE
Possible to create two different sets of prefs?

### DISCUSSION THREAD

**Discussion by fatrabbit-la** [artisan-roaster-scope/artisan#946] [Q&A]
I have a single set of preferences I use for roasting coffee that has the phases, alarms, etc. just the way I like it and this is the default every time I open the app. I'm about to venture into roasting chocolate on the same machine and would like to know if there's a way I can create a separate set of preferences for this that has its own unique phases, alarms, etc. in such a way that I can easily switch back and forth.

**Comment by MAKOMO** [comment]
Just save your first settings to a file via menu Help >> Save Settings. Define your second set of settings and save them to a different file, again via Help >> Save Settings. You can quickly load each of those via the menu Help >> Load Recent Settings.
